releasenotes/notes/flask_improve_resource_naming_when_error_occurs-ea094145551e4ec7.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix resource naming of Flask request span when errors occur in middleware.
```

```suggestion
    flask: Fix resource naming of request span when errors occur in middleware.
```

I think @Kyle-Verhoog will make this suggestion anyways.